pkg/kopia/utils.go

```diff
@@ -32,14 +32,16 @@ import (
 	"github.com/pkg/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
+
+	snap "github.com/kanisterio/kanister/pkg/kopia/snapshot"
 )
 
 const (
-	// defaultConfigFilePath is the file which contains kopia repo config
-	defaultConfigFilePath = "/tmp/kopia-repository.config"
+	// DefaultClientConfigFilePath is the file which contains kopia repo config
+	DefaultClientConfigFilePath = "/tmp/kopia-repository.config"
 
-	// defaultCacheDirectory is the directory where kopia content cache is created
-	defaultCacheDirectory = "/tmp/kopia-cache"
+	// DefaultClientCacheDirectory is the directory where kopia content cache is created
+	DefaultClientCacheDirectory = "/tmp/kopia-cache"
```

Comment:
Will this be moved to `const.go` in a future PR?

Reply:
Yes. That's moved in a `const.go` in PR B